docs/topics/forms/formsets.txt

```diff
@@ -448,6 +448,47 @@ happen when the user changes these values::
     {'pub_date': datetime.date(2008, 5, 11), 'ORDER': 1, 'title': 'Article #2'}
     {'pub_date': datetime.date(2008, 5, 10), 'ORDER': 2, 'title': 'Article #1'}
 
+``ordering_widget``
+-------------------
+
+.. attribute:: BaseFormSet.ordering_widget
+
+Default: :class:`~django.forms.NumberInput`
+
+Lets you change the order field widget::
+
+    class FormSet(BaseFormSet):
+        ordering_widget = HiddenInput
+
+    class ArticleForm(ModelForm):
+        class Meta:
+            model = Article
+            fields = ('content', )
+
+    ArticleFormSet = formset_factory(ArticleForm, formset=FormSet, can_order=True)
+
+.. versionadded:: 3.0
+
+``ordering_widget``
```

Comment:
This looks wrong. `get_ordering_widget()` no?

Reply:
yes will fix it